src/plugins/intel_gpu/src/graph/impls/onednn/utils.cpp

```diff
@@ -443,6 +465,20 @@ cldnn::format find_data_format(dnnl::memory::desc desc) {
         }
     }
 
+    if (desc.get_ndims() == 3 && desc.get_inner_nblks() == 0) {
+        // Special case for 3D tensors without blocking
+        // This is a common case for 2D tensors with batch dimension
```

Comment:
what does `2d tensors with batch dimension` mean?

Reply:
I don't know. I will remove this comment.